src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const сalculator = {
+    result: 0,
+
+    add(result, value) {
+      return result + value;
+    },
+
+    subtract(result, value) {
+      return result - value;
+    },
+
+    divide(result, value) {
+      return result / value;
+    },
+
+    multiply(result, value) {
+      return result * value;
+    },
+
+    operate(callback, value) {
+      this.result = callback(this.result, value);
```

Comment:
Consider binding the context for the function call

Reply:
Done ✅